dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCClient.scala

```diff
@@ -93,8 +58,8 @@ case class BinaryOutcomeDLCClient(
     losePayout: CurrencyUnit,
     timeouts: DLCTimeouts,
     feeRate: FeeUnit,
-    changeSPK: WitnessScriptPubKeyV0,
-    remoteChangeSPK: WitnessScriptPubKeyV0,
+    changeSPK: WitnessScriptPubKey,
+    remoteChangeSPK: WitnessScriptPubKey,
```

Comment:
Is there a specific reason for allowing non-V0 here? I'm not necessarily against it, just curious